src/wp-includes/post.php

```diff
@@ -7795,6 +7796,31 @@ function _prime_post_caches( $ids, $update_term_cache = true, $update_meta_cache
 	}
 }
 
+/**
+ * Prime post parent caches.
+ *
+ * @global wpdb $wpdb WordPress database abstraction object.
+ *
+ * @param int[] $ids               ID list.
+ */
+function _prime_post_parents_caches( $ids ) {
```

Comment:
Might want a better name for this.

Reply:
There is no test coverage for this function ATM. But it is worth noting that `_prime_post_caches` doesnt seem to have any test coverage ATM either.

Reply:
Adding test coverage for both functions would be nice to have, but particularly this new one. Testing that the correct parent posts are primed and that the cache is correctly cleaned when caches are flushed or when the parent post for a post changes, would be sufficient. I'm not sure if the new tests you've added fully cover the use cases I've mentioned, but seems like they may cover most of it already—in which case adding an `@covers` annotation to those tests for `_prime_post_caches` could be appropriate.

Also, FWIW, I think the current name is fine.